change/@fluentui-react-provider-c53fd31c-6213-4dc3-a622-f036b04f8d5a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "deprecate unused useListItemButtonStyles_unstable",
```

Comment:
is this file necessary? i don't see changes in the provider

Reply:
you're correct, I reverted the change (which was removing one line) and deprecated the API instead with a comment. This change is probably obsolete and I'll remove it.